solutions/USACO dec15-speeding-ticket.mdx

```diff
@@ -77,26 +30,26 @@ int main(){
 
 	int start = 0;
 	array<int,100> limit;
```

Comment:
why arr<int, 100> ??? Just make it vector< int >limit(100).

Reply:
i think it's better to use an array than a vector when the object is constant sized.

Reply:
In general I'd say if it's not bad practice or unclear let them do what they want. Though I should also improve... I tend to make suggestions purely based on preference as well.

Reply:
> In general I'd say if it's not bad practice or unclear let them do what they want. Though I should also improve... I tend to make suggestions purely based on preference as well.

Likewise. I generally prefer vectors LOL.

Reply:
Since it's constant sized can you make it `int limit[100]`?